3.0.42.0/README.md

```diff
@@ -0,0 +1,142 @@
+   available parameters.
+1. Run the following command to generate secrets, certificates, and other
+   required configurations:
+   ```bash
+   ./scripts/setup.sh -f <path-to-the-values-yaml-file> -e <central endpoint>
+   ```
+   - You must specify the path to the `values.yaml` file with `-f` parameter when
+     you run the setup script. If you don't specify `values.yaml` file, the
+     setup script uses the `values.yaml` file from the chart directory.
+1. Run the Helm chart by using the following command:
+   
+   - For Helm v2:
+     ```
+     helm install --name sensor <path-to-the-directory-containing-the-charts> --namespace stackrox
+     ```
+   - For Helm v3:
+     ```
+     helm install sensor <path-to-the-directory-containing-the-charts> --namespace stackrox
+     ```
+   > **NOTE**
+   > - You can't use the `-f` parameter to specify the `values.yaml` file when you run the install command.
+   > - You must specify the namespace as `stackrox` by using the `--namespace stackrox` option with each helm command.
+
+## Uninstall Sensor using Helm chart
+
+1. Run the Helm chart by using the following command:
+   
+   - For Helm v2:
+     ```
+     helm delete --name sensor --namespace stackrox
+     ```
+   - For Helm v3:
+     ```
+     helm delete sensor --namespace stackrox
+     ```
+1. To verify if the Sensor is uninstalled, view the output of the following
+   command:
+   ```
+   helm list --namespace stackrox
+   ```
+
+## Upgrade Sensor using Helm chart
+
+> **NOTE**
+>
+> - You can only upgrade those Sensor installations that you've [installed using Helm charts](#install-sensor-using-helm-chart).
+> - You don't have to run the setup script when you upgrade.
+
+1. Navigate to the [stackrox/helm-charts](https://github.com/stackrox/helm-charts)
+   repository, and download the helm charts folder that corresponds to the
+   StackRox Kubernetes Security Platform version you are using. For example, if
+   you are using the StackRox Kubernetes Security Platform version 3.0.41,
+   download the folder named `3.0.41`.
+1. From the downloaded folder, modify the `values.yaml` file based on your
+   environment. See the [Configuration](#configuration) section to understand the
+   available parameters.
+1. Run the Helm upgrade command.
+   - For Helm v2:
+     ```
+     helm upgrade --namespace stackrox --name sensor .
+     ```
+   - For Helm v3:
+     ```
+     helm upgrade --namespace stackrox sensor .
+     ```
+1. Run the following commands in every secured cluster in which you have enabled
+   the admission controller.
+   - For Kubernetes:
+     ```
+     kubectl -n stackrox patch deploy/admission-control -p '{"spec":{"template":{"spec":{"containers":[{"name": "admission-control","readinessProbe":  {"initialDelaySeconds": 5, "periodSeconds": 5, "failureThreshold": 1}}]}}}}'
+     ```
+   - For OpenShift:
+     ```
+     oc -n stackrox patch scc sensor -p '{"priority": 0}'
+     oc -n stackrox patch scc central -p '{"priority": 0}'
```

Comment:
I am assuming if they are on OpenShift, they are using Central on OpenShift and not a mixed environment? Let me know if we need to specifically say `for cluster running central on Openshift`.